application/common/core.cpp

```diff
@@ -295,13 +337,40 @@ main(int argc, char *argv[])
 
     /* Main render loop */
     gLoomExecutionDone = 0;
+
+    /* Game Controller stuff */
+    SDL_GameControllerEventState(SDL_ENABLE);
+
+    //Adding non-existent controller mappings
+    int mappingsAdded = SDL_GameControllerAddMapping("4d6963726f736f66742050432d6a6f79,X360 Controller,platform:Windows,a:b10,b:b11,x:b12,y:b13,back:b5,start:b4,guide:b14,leftshoulder:b8,rightshoulder:b9,leftstick:b6,rightstick:b7,leftx:a0,lefty:a1,rightx:a2,righty:a3,lefttrigger:a4,righttrigger:a5,dpup:b0,dpleft:b2,dpdown:b1,dpright:b3,");
+    //int mappingsAdded = SDL_GameControllerAddMappingsFromFile("gamecontrollerdb.txt");
+    if (mappingsAdded >= 0)
+    {
+        lmLog(coreLogGroup, "Succesfully added %d mappings", mappingsAdded);
+    }
+    else
+    {
+        lmLog(coreLogGroup, "SDL ERROR: %s", SDL_GetError());
+    }
+
+    SDL_version compiled;
+    SDL_version linked;
+
+    SDL_VERSION(&compiled);
+    SDL_GetVersion(&linked);
+
+    lmLog(coreLogGroup, "Compiled with SDL version %d.%d.%d ...", compiled.major, compiled.minor, compiled.patch);
```

Comment:
I think both should be debug priority. I'm not even sure if we should keep SDL version in or not yet. Would appreciate some input.